src/Kestrel.Core/Internal/Infrastructure/HttpCharacters.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Infrastructure
+{
+    internal static class HttpCharacters
+    {
+        public static readonly int TableSize = 128;
+        public static readonly bool[] AlphaNumeric;
```

Comment:
Not used outside of the cctor, no need for the static field.

Reply:
It's a static ctor so these need to be static as well